Automattic/jetpack
# My Jetpack: Improve Layout structure

Fixes #22678 

#### Changes proposed in this Pull Request:

* Update `Layout` structure using `Container` and `Col`.
* Remove `Container` and `Col` from `ConnectScreen`.
* Update `ProductInterstitial` to use `fluid` and remove `wrapper`.
* Move `Router` handling to `MyJetpack` component.

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Build `My Jetpack`(`jetpack build packages/my-jetpack`)
* Navigate to `connection`, `add-bost` and `add-search` routes.
* You should see pages working properly.

#### Demo

![Screen Shot 2022-02-09 at 10 54 49](https://user-images.githubusercontent.com/1663717/153216859-75e3cde5-b32c-4d4c-92ee-a0939b6fcdd3.png)
![Screen Shot 2022-02-09 at 10 55 43](https://user-images.githubusercontent.com/1663717/153216871-9d62e9d9-7404-4ff1-8041-dd3ed92d52fe.png)
![Screen Shot 2022-02-09 at 10 55 53](https://user-images.githubusercontent.com/1663717/153216881-0ef122af-62c7-4b67-8165-331d33d34885.png)
